package.json

```diff
@@ -37,6 +38,7 @@
     "@types/react": "18.2.4",
     "@types/react-dom": "18.2.3",
     "@types/react-slick": "^0.23.10",
+    "@vercel/kv": "^0.2.1",
```

Comment:
Didn't we remove KV? Why are we adding it into the package json if it's no longer used?